packages/client/tests/tests-mpe-utils.ts

```diff
@@ -75,7 +75,9 @@ export async function createMpeRoom(): Promise<{
         1,
     );
 
-    const goToLibraryButton = screen.getByText(/library/i);
+    const goToLibraryButton = (
```

Comment:
I have no idea but this locator will fail during client tests in the ci, resolving to multiple elements :thinking: 
It's should not be possible to duplicate the bottomBar Icons, no issues in local

Reply:
Strange 😮